.circleci/config.yml

```diff
@@ -74,6 +74,33 @@ jobs:
       - store_artifacts:
           path: /tmp/smoketest.log
 
+  samplegen_smoke_test:
+    working_directory: /usr/src/artman
+    docker:
+      - image: docker:17.05.0-ce-git
+    steps:
+      - *checkout_artman
+      - setup_remote_docker
+      - *restore_artman_cache
+      - *load_build_and_save_artman_image
+      - *save_artman_cache
+      - run:
+          name: Check samples are generated for selected APIs
+          command: |
+            EXIT_STATUS=0
+            docker run -it --rm -d --name samplegen --workdir /tmp -e RUNNING_IN_ARTMAN_DOCKER=True artman
+            docker exec samplegen mkdir /tmp/language-python
+            docker exec samplegen artman --local --output-dir /tmp/language-python --config=google/cloud/language/artman_language_v1.yaml --root-dir=/googleapis --generator-args='--dev_samples' generate python_gapic
+            docker exec samplegen find /tmp/language-python/python/language-v1/samples/v1 -mindepth 1 || EXIT_STATUS=$?
```

Comment:
Here you only check that the `samples/v1` folder is generated, but you don't actually check that any files are there. It's better than no check at all, but I would probably check that the number of Python files inside is greater than zero (same for PHP and other language).